magicctapipe/scripts/lst1_magic/semi_automatic_scripts/job_accounting.py

```diff
@@ -96,19 +96,29 @@ def main():
         source_out = "*"
 
     indir = f"{work_dir}/v{args.version}/{source_out}/{args.data_level}"
-    dirs = sorted(
-        glob.glob(f"{indir}/[0-9]*/[M0-9]*")
-        + glob.glob(f"{indir}/Merged_[0-9]*")
-        + glob.glob(f"{indir}/" + "[0-9]" * 8)
-    )
+
+    if args.data_level == "MergedStereo":
+        dirs = sorted(
+            glob.glob(
+                f"{work_dir}/v{args.version}/{source_out}/DL1Stereo/[0-9]*/Merged"
```

Comment:
I think this code can be made much simpler if we keep the same structure for DL1 merged files and DL1Stereo merged files.
I.e. when we merged the DL1 files the output is DL1/Merged/dates...
and for DL1Stereo you did DL1Stereo/date/Merged/...
this way we can avoid special cases and simply pass "DL1Stereo/Merged" as the data_level

Reply:
Actually yes. I will fix it (hopefully) today